paasta_tools/kubernetes_tools.py

```diff
@@ -1617,13 +1617,20 @@ def format_kubernetes_app(self) -> Union[V1Deployment, V1StatefulSet]:
                     ),
                 )
             else:
+                # Default to none for progress_deadline_seconds to not change
+                # existing deployments
+                progress_deadline_seconds = None
+                if self.get_min_task_uptime() >= 600:
```

Comment:
might be nice to have a named constant for 600 - then we can skip the comment explaining the magic number :)